app/src/main/java/com/duckduckgo/app/sync/FakeDeviceSyncState.kt

```diff
@@ -26,8 +26,7 @@ import javax.inject.*
  */
 @ContributesBinding(
     scope = AppScope::class,
-    // mitigation for https://app.asana.com/0/414730916066338/1204248963638410/f
-    priority = ContributesBinding.Priority.HIGHEST,
+    priority = ContributesBinding.Priority.NORMAL,
```

Comment:
Why do we need the `FakeDeviceSyncState`. Can we remove it and use `AppBuildConfig` in `AppDeviceSyncState` to enable it only for internal builds?

Reply:
Currently, we use `internalImplementation` for sync modules. We need this fake so we can compile other flavors.

That being said, even though I'm not confident, we will move to `implementation` this week, and test if that causes any issues when releasing into other stores. If no issues, we will remove the fake class.